backend/src/main/java/gov/cdc/usds/simplereport/db/model/TestOrder.java

```diff
@@ -35,7 +35,7 @@
 	@JoinColumn(name = "device_type_id")
 	private DeviceType deviceType;
 	@Column
-	private LocalDate dateTested;
+	private Date dateTested;
```

Comment:
It's not clear to me that this column should still exist—should we instead rely on the `TestEvent` join?

Reply:
That is a great point! Will update to us `TestEvent`. Which is also recording the time  in `TestEvent.created_at` :D

Reply:
Is there a reason for using Date rather than something like `OffsetDateTime`, which encodes the time zone by default?

Reply:
My view is that the database should hold raw timestamp data, in milliseconds since whatever the relevant epoch is, and let other people worry about time zones. This runs into problems if your database thinks it is in a different time zone than it actually is, but I would rather count on the database OS being correctly configured than dicker around with time zones in the persistence layer.

(Postgresql does have a `TIMESTAMP WITH TIME ZONE` type, so we could do that if we wanted to, but each time I return to this problem I find myself less inclined to use it.)

I have previously proposed that we take time zone as an argument in the graphql query, and render times into the appropriate time zone at runtime: this offer was roundly rejected last time I brought it up, on the grounds that Javascript is perfectly capable of parsing and translating an ISO timestamp string. 😄